lib/sles4sap/console_redirection.pm

```diff
@@ -102,14 +111,9 @@ sub connect_target_to_serial {
     croak "IP address '$args{destination_ip}' is not valid." unless grep(/^$RE{net}{IPv4}$/, $args{destination_ip});
     croak 'Global variable "$serialdev" undefined' unless $serialdev;
 
-    # This captures initial machine id which serves as a baseline host.
-    # disconnect_target_from_serial() types 'exit + enter' in a loop until /etc/machine-id matches machine id defined
-    # by BASE_VM_ID. This means original host before redirection was reached.
-    set_var('BASE_VM_ID', script_output 'cat /etc/machine-id');
-
     if (check_serial_redirection()) {
         record_info('Redirect ON', "Console is already redirected to:" . script_output('hostname', quiet => 1));
-        return;
+        return 1;
```

Comment:
maybe drop one sentence about the return value in the function documentation

Reply:
Thanks. Done.